### PR TITLE
sprint funding draft

### PR DESCRIPTION
**Pull request by hpk42**
a draft so we go for funding.  What is missing:

- deadline for providing funding.  Mid march?
 
- which platform to use?  I'd think indiegogo but am open to other suggestions. We can also do a paypal, bitcoin address and i mention the invoicing to merlinux possibility  i.e. we don't need to just use one platform.  I plan to contact some companies where i had previous testing related consultings/trainings with.

### CODE REVIEW

**Comment by hpk42** [comment]
oh, and feel free to contribute new topics or just directly commit them after this PR is merged.

File: doc/en/announce/sprint-funding-2016.txt [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+which also means we are going to do some hiking likely.
+
+Here are preliminary participants who said they are likely to come,
+given some expenses funding::
+
+    Andreas Pelme, Sweden
+    Brianna Laughner, Australia
+    Bruno Oliveira, Brazil
+    Florian Bruhin, Switzerland
+    Florian Bruynooghe, UK
```
Comment (flub) [inline doc/en/announce/sprint-funding-2016.txt]: Floris :-)

**Comment by nicoddemus** [comment]
LGTM! :smile:

File: doc/en/announce/sprint-funding-2016.txt (hunk 1 above)
Comment (RonnyPfannschmidt) [inline doc/en/announce/sprint-funding-2016.txt]: details on my attendance will be cleared next week

**Comment by RonnyPfannschmidt** [comment]
LGTM :)

**Pull request merged**